### ISSUE TITLE
the mig-nat-http-lb outputs.tf file require a sensitive data

### ISSUE THREAD

**Issue by anadiegues** [terraform-google-modules/terraform-google-lb-http#391]
### TL;DR

the example mig-nat-http-lb outputs.tf entrance for "backend_services" need to setup sensite =true

### Observed behavior

Error: Output refers to sensitive values
│ 
│   on outputs.tf line 26:
│   26: output "backend_services" {
│ 
│ To reduce the risk of accidentally exporting sensitive data that was intended to be only internal, Terraform requires that any root module output containing sensitive data be explicitly marked as sensitive, to confirm your
│ intent.
│ 
│ If you do intend to export this data, annotate the output value as sensitive by adding the following argument:
│     sensitive = true

### Terraform Configuration

```hcl
output "load-balancer-ip" {
  value = module.gce-lb-http.external_ip
}

output "load-balancer-ipv6" {
  value       = module.gce-lb-http.ipv6_enabled ? module.gce-lb-http.external_ipv6_address : "undefined"
  description = "The IPv6 address of the load-balancer, if enabled; else \"undefined\""
}

output "backend_services" {
  value = module.gce-lb-http.backend_services
}
```

### Terraform Version

```sh
terraform --version
Terraform v1.6.5
on linux_amd64
```

### Additional information

corrected outputs.tf file:

output "load-balancer-ip" {
  value = module.gce-lb-http.external_ip
}

output "load-balancer-ipv6" {
  value       = module.gce-lb-http.ipv6_enabled ? module.gce-lb-http.external_ipv6_address : "undefined"
  description = "The IPv6 address of the load-balancer, if enabled; else \"undefined\""
}

output "backend_services" {
  value = module.gce-lb-http.backend_services
  sensitive = true
}
